trusted-committer/de/04-uplevelling-community-members.asciidoc

```diff
@@ -0,0 +1,23 @@
+
+[[upleveling]]
+== Weiterbildung von Community Mitgliedern
+
+Es gibt ein Kontinuum bei der Teilnahme an einer InnerSource Community. Es gibt Menschen, denen die Community nicht bekannt ist. Neulinge könnten sich für die Community und deren Produkt interessieren, haben aber bisher noch nicht damit gearbeitet. _Consumer_ nutzen die Software, haben aber bisher noch nicht selbst dazu beigetragen.
+Dann gibt es die https://innersourcecommons.org/resources/learningpath/contributor/index[_Contributors_], die schon mindestens einen Beitrag geleistet haben, und schließlich die _Trusted Committer_, welche Verantwortung sowohl für die Software als auch die Community übernehmen.
+Als ein _Trusted Committer_ bist du verantwortlich, Individuen innerhalb dieses Kontinuums zu bewegen und deren Fähigkeiten zu verbessern, Beiträge zu liefern.
+In diesem Sinne agieren _Trusted Committer_ als Multiplikatoren in ihrer Community.
+
+Es ist wichtig für _Trusted Committer_ ihr Produkt und ihre Community zu vermarkten, um weitere Neueinsteiger und _Consumer_ der Community zu werben.
+Sie sollten also Gelegentheiten kommunizieren, eigene Beiträge für _Consumer_ zu erstellen und versuchen, die Interessen von möglichen https://innersourcecommons.org/resources/learningpath/contributor/index[_Contributors_] zu ermitteln und mit denen der Community in Einklang zu bringen.
+Als gute Praxis hat sich dabei bewährt, wenn https://innersourcecommons.org/resources/learningpath/contributor/index[_Contributors_] an etwas arbeiten können, was ihnen bei ihrer täglichen Arbeit oder ihrer Rolle im Unternehmen hilft. Gute Beispiele dafür sind Entwicklungswerkzeuge oder die Automatisierung von Abläufen.
+
+Letztendlich ist der _Trusted Committer_ dafür verantwortlich, https://innersourcecommons.org/resources/learningpath/contributor/index[_Contributors_] mit dem Potential zu wachsen zu indentifizieren und zu unterstützen, indem er sie ermutigt, herausfordernde Aufgaben zu übernehmen und sie dabei bis ins Ziel unterstützt.
```

Comment:
We can have multiple TCs and the rest of the text can be read as adressing multiple TCs. Hence, here's a commit suggestion.
```suggestion
Letztendlich sind die _Trusted Committers_ dafür verantwortlich, https://innersourcecommons.org/resources/learningpath/contributor/index[_Contributors_] mit dem Potential zu wachsen zu indentifizieren und zu unterstützen, indem sie sie ermutigen, herausfordernde Aufgaben zu übernehmen und sie dabei bis ins Ziel unterstützen.
```